web/src/components/pages/contribution/speak/audio-web.ts

```diff
@@ -1,6 +1,6 @@
-import { isNativeIOS } from '../../../../utility';
+import { isNativeIOS, isIOS } from '../../../../utility';
 
-const AUDIO_TYPE = 'audio/ogg; codecs=opus';
+const AUDIO_TYPE = `audio/${isIOS() ? 'wav' : 'ogg'}; codecs=opus`;
```

Comment:
Note: as [detailed in `audio-recorder-polyfill`'s documentation](https://github.com/ai/audio-recorder-polyfill#custom-encoder), it's possible to use a custom encoder to enable `ogg` encoding on iOS. I think using `wav` here is more straightforward, and simplifies immediate playback on the device. And we transcode the audio to `mp3` on the server anyway.

If you have any concerns or would prefer we keep everything as `ogg` client-side, let me know and I'll look into it!

Reply:
I'm curious about if there's a filesize difference in mp3s generated from wav vs. ogg, because my main concern would be if wav is significantly bigger because it compresses less, the mp3 will be too... There's not really an easy way of testing this, except maybe by recording clips on iOS from a specific account on staging and then examining the subsequent clips in AWS? Thoughts?

Reply:
Hmm, good question. We can also use `mp3`s directly here, the polyfill comes bundled with an encoder. Do you think that would be better?

Reply:
We discussed this offline and are going to stick with .wav for now 👍